doc/guides/building-node-with-ninja.md

```diff
@@ -0,0 +1,41 @@
+# Building Node with Ninja
+
+The purpose of this guide is to show how to build Node.js using [Ninja][], as doing so can be much quicker than using `make`. Please see [Ninja's site][Ninja] for installation instructions (unix only).
+
+To build Node with ninja, there are 4 steps that must be taken:
+1. Configure the project via `./configure` as usual.
+2. Make `tools/gyp_node.py` produce Ninja output.
+3. Run Ninja in `out/Release` to produce release output.
+4. Link `out/Release/node` to `./node`.
+
+Those 4 steps can be achieved with the following commands:
```

Comment:
These 4 commands might be better placed as part of each of the previous 4 items, to avoid confusion, and to avoid making it look, at first glance, like there are two *separate* sets of instructions that need to be followed.